src/Internal/Metadata/Cip25/V2.purs

```diff
@@ -149,20 +151,20 @@ instance DecodeAeson Cip25V2 where
 -- the standard only specifies the encoding for Cip25Metadata).
 metadataEntryToMetadata :: Cip25MetadataEntry -> TransactionMetadatum
 metadataEntryToMetadata (Cip25MetadataEntry entry) = toMetadata $
-  [ "name" /\ anyToMetadata entry.name
```

Comment:
I'm curious about the impact of switching from `anyToMetadata` to `toMetadata`. What was your motivation for doing so?

In my own investigation we never use the interesting case of `anyToMetadata` (I attached an impossible constraint to the first instance), does that mean we can throw away the whole type + class @klntsky?

Reply:
I think AnyToMetadata is over-engineered. My vote is to remove it if we can.